setup.py

```diff
@@ -72,7 +73,7 @@ def run(self):
     "treelib>=1.5.5",
     "inflect>=2.1.0",
     "humanize>=0.5.1",
-    "PyYAML>=5.1.2",
+    "PyYAML==5.1.2",  # Compatibility with awscli requiring (< 5.2)
```

Comment:
Feels weird to freeze PyYAML for the main project because of the test requirements. Maybe let's at least put a range there, same as in awscli https://github.com/aws/aws-cli/blob/develop/setup.py#L48 ?

Reply:
:+1: